sql/internal/migrate/client.go

```diff
@@ -0,0 +1,212 @@
+// Copyright 2021-present The Atlas Authors. All rights reserved.
+// This source code is licensed under the Apache 2.0 license found
+// in the LICENSE file in the root directory of this source tree.
+
+// Code generated by entc, DO NOT EDIT.
+
+package migrate
```

Comment:
Can you please use the standard ent naming? You can place it also under the `sql/migrate` package.